src/shared/Git-Credential-Manager.UI.Avalonia/Program.cs

```diff
@@ -44,7 +45,8 @@ private static void AppMain(object o)
 
             string appPath = ApplicationBase.GetEntryApplicationPath();
             string installDir = ApplicationBase.GetInstallationDirectory();
-            using (var context = new CommandContext(appPath, installDir))
+            // Pass in a "dummy" stopwatch, since TRACE2 is not enabled for helper exes.
```

Comment:
Any reason why not to implement Trace2 for helper UI? Or is this a 'todo' in the future thing?

Reply:
My reasoning was that I didn't see a lot of `TRACE` stuff in the helper UI code. Could definitely be a future thing though.